lona/app.py

```diff
@@ -157,7 +207,7 @@ def add_static_file(self, name, string='', path=''):
         )
 
     # server ##################################################################
-    def run(self, **args):
+    def run(self, **args: Any) -> None:  # TODO: change to explicit arguments
```

Comment:
Does this mean your pr is not ready, or do you want to add this todo string?

Reply:
Removed comment